extensions/machine-learning-services/src/common/utils.ts

```diff
@@ -47,6 +47,20 @@ export function getPythonExePath(rootFolder: string): string {
 		process.platform === constants.winPlatform ? 'python.exe' : 'bin/python3');
 }
 
+export function getRSqlMlUtilsPath(rootFolder: string): string {
+	return path.join(
+		rootFolder,
+		'packages',
+		'sqlmlutils_0.7.1.zip');
+}
```

Comment:
This is the second place this is defined - putting these in a config file as I suggested seems like a good idea here

Reply:
for now the name is hard coded. I need to find a better way for it

---
In reply to: [366614655](https://github.com/microsoft/azuredatastudio/pull/8870#discussion_r366614655) [](ancestors = 366614655)